bindings/python/PythonLoader.cpp

```diff
@@ -0,0 +1,129 @@
+#include <Python.h>
+#include "gameobject.h"
+#include "swigruntime.py.h"
+#include "scripts/PythonLoader.h"
+#include "script.h"
+#include <string>
+#include <algorithm>
+using std::string;
+using nautical::GameObject;
+using nautical::script::Script;
+
+namespace
+{
+    void addPythonModulePath(string p)
+    {
+        static char pthc[5] = "path";
+        PyObject *sys_path = PySys_GetObject(pthc);
+        if (sys_path == NULL)
+            return;
+        PyObject *path = PyString_FromString(p.c_str());
+        if (path != NULL)
+        {
+            PyList_Append(sys_path, path);
+            Py_DECREF(path);
+        }
+    }
+
+    struct EndsWith
+    {
+        EndsWith(string str) : str(str)
+        {
+        }
+        bool operator()(string prefix)
+        {
+            return (str.substr(str.length() - prefix.length()) != prefix);
+        }
+
+    private:
+        string str;
+    };
+
+    class PythonScript : public Script
+    {
+    public:
+        PythonScript(PyObject *obj, GameObject* gameObject) : _obj{obj}
+        {
+            auto type_info = SWIG_TypeQuery("nautical::GameObject *");
+            auto pyObject = SWIG_NewPointerObj(gameObject, type_info, 0);
+            int ret = PyObject_SetAttrString(obj, "gameobject", pyObject);
+        }
+        ~PythonScript()
```

Comment:
Should we clear the reference to `_obj` here. I think `PY_DECREF` would be the necessary function?